xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml

```diff
@@ -1203,22 +1203,22 @@ Setting cpu: 2
       <title>Test workload: STREAM</title>
 
       <para><emphasis role="italic">STREAM</emphasis> is a memory bandwidth benchmark
-        created by Dr.  John D. McCalpin from the University of Virginia (for more
+        created by Dr. John D. McCalpin from the University of Virginia (for more
         information, see <link xlink:href="https://www.cs.virginia.edu/stream/"
         >https://www.cs.virginia.edu/stream/</link>). It can be used to measure bandwidth
         of each cache level and bandwidth to main memory while calculating four basic
-        vector operations.  Each operation can exhibit different throughputs to main memory
+        vector operations. Each operation can exhibit different throughputs to main memory
         depending on the locations and type of access.</para>
 
       <para>The benchmark was configured to run both single-threaded and parallelized with
-	OpenMP to take advantage of multiple memory controllers. The array elements for
-	the benchmark was set at 268,435,456 elements at compile time so each that array
+	OpenMP to take advantage of multiple memory controllers. The array of elements for
```

Comment:
`The array elements for the benchmark were set at 268,435,456 elements at compile time`

array element is a fix term